portfolio/src/main/webapp/script.js

```diff
@@ -45,4 +45,12 @@ async function showServerTime() {
   
     const dateContainer = document.getElementById('date-container');
     dateContainer.innerText = textFromResponse;
-  }
+}
+
+async function getServerGreeting() {
+    
+    const responseFromServer = await fetch('/greeting');
```

Comment:
I thought I left a comment about this, but I would think about renaming this to serverResponse, it's shorter and still clear without "from".

Reply:
Updated